### ISSUE TITLE
activate fails with UNAVAIL pools

### ISSUE THREAD

**Issue by rwestlund** [iocage/iocage#206]
Using HEAD (cd9464527b3876157461631ea7ac3c89382b4de2):

Given the following pools:
```
NAME       SIZE  ALLOC   FREE  EXPANDSZ   FRAG    CAP  DEDUP  HEALTH  ALTROOT
cryopool      -      -      -         -      -      -      -  UNAVAIL  -
zroot     10.9T   465G  10.4T         -     4%     4%  1.00x  ONLINE  -
```

Trying to activate iocage fails:
```
root@r2d2 /u/p/s/py3-iocage# iocage activate zroot
Traceback (most recent call last):
  File "/usr/local/bin/iocage", line 10, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/iocage/cli/activate.py", line 37, in cli
    ioc.IOCage(activate=True).activate(zpool)
  File "/usr/local/lib/python3.6/site-packages/iocage/lib/iocage.py", line 310, in activate
    ds = self.zfs.get_dataset(pool.name)
  File "libzfs.pyx", line 566, in libzfs.ZFS.get_dataset (libzfs.c:18784)
libzfs.ZFSException: Dataset cryopool not found
```
This should probably exclude pools that are `UNAVAIL`, perhaps with a warning:
https://github.com/iocage/iocage/blob/cd9464527b3876157461631ea7ac3c89382b4de2/iocage/lib/iocage.py#L293

**Comment by rwestlund** [comment]
@skarekrow you fixed a related bug, but this one is still there in current HEAD (25727df28a8711d4f8dd413f912f04a1f2a36dc0).  This crash happens when iterating over the pools, to set each one active or inactive.  Even when the `UNAVAIL` pool is not the target pool, you try to set `org.freebsd.ioc:active=no` on it.

Given the following pools:
```
NAME       SIZE  ALLOC   FREE  EXPANDSZ   FRAG    CAP  DEDUP  HEALTH  ALTROOT
atest     1008M    82K  1008M         -     0%     0%  1.00x  UNAVAIL  -
cryopool  1.98T   639G  1.36T         -    21%    31%  1.00x  ONLINE  -
zroot     10.9T   466G  10.4T         -     4%     4%  1.00x  ONLINE  -
```
```
root@r2d2 /u/h/randy# iocage activate zroot
Traceback (most recent call last):
  File "/usr/local/bin/iocage", line 10, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/iocage/cli/activate.py", line 37, in cli
    ioc.IOCage(activate=True).activate(zpool)
  File "/usr/local/lib/python3.6/site-packages/iocage/lib/iocage.py", line 320, in activate
    ds = self.zfs.get_dataset(pool.name)
  File "libzfs.pyx", line 566, in libzfs.ZFS.get_dataset (libzfs.c:18784)
libzfs.ZFSException: Dataset atest not found
```

**Comment by skarekrow** [comment]
Bah! Fixed in 42fb465acf67b65a0ce9299e64c00c586c4c8899

**Comment by rwestlund** [comment]
@skarekrow it works now, thanks :)
